### DISCUSSION TITLE
Document the .env

### DISCUSSION THREAD

**Discussion by ghost** [KRTirtho/spotube#594] [Ideas]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Pain

Idk wth is needed here
SUPABASE_URL=
SUPABASE_API_KEY=

The format:
 SPOTIFY_SECRETS=clintId1:clientSecret1,clientId2:clientSecret2
SPOTIFY_SECRETS=

If I am being an idiot for asking this, I would like to know the reasons, please XD im not a dev

### Suggested solution

Just add it to the build instrucions

**Comment by KRTirtho** [comment]
Check the `.env.example` file it contains all necessary keys

**Comment by KRTirtho** [comment]
> Idk wth is needed here
> SUPABASE_URL=
> SUPABASE_API_KEY=

You don't need to fill this locally. Spotube can work without them. Just pass an empty string

> The format:
> SPOTIFY_SECRETS=clintId1:clientSecret1,clientId2:clientSecret2
> SPOTIFY_SECRETS=

> If I am being an idiot for asking this, I would like to know the reasons, please XD im not a dev

You need your own Spotify client id and client secret. Just go to https://developers.spotify.com and create an app
Then copy paste it as

```
SPOTIFY_SECRETS=<client-id>:<client-secret>
```

Also follow the, [CONTRIBUTION.md](https://github.com/KRTirtho/spotube/blob/master/CONTRIBUTION.md) to see how to build the app from source
